public/javascripts/SVLabel/src/SVLabel/Main.js

```diff
@@ -176,6 +176,13 @@ function Main (params) {
         }
         svl.popUpMessage = new PopUpMessage(svl.form, svl.storage, svl.taskContainer, svl.tracker, svl.user, svl.onboardingModel, svl.ui.popUpMessage);
 
+        // Logs when the webpage's visiblity changes.
```

Comment:
When you first add the event listeners, can you first do a check for whether the page is in/out of focus and log that?